web/src/lib/domain-logic/isLicenseDataFromDatabaseDataMoreRecent.test.ts

```diff
@@ -0,0 +1,80 @@
+import { isLicenseDataFromDatabaseDataMoreRecent } from "@/lib/domain-logic/isLicenseDataFromDatabaseDataMoreRecent";
+import { generateBusiness, generateLicenseData, getCurrentDate } from "@businessnjgovnavigator/shared/";
+
+const currentDate = getCurrentDate();
+const currentDateISOString = currentDate.toISOString();
+const currentDatePlusOneHourISOString = currentDate.add(1, "hour").toISOString();
+
+describe("isLicenseDataFromDatabaseDataMoreRecent", () => {
```

Comment:
[pebble] all of the expect statements should be explicitly checking `expect(result).toBe(true) or ..toBe(false)` since the function is explicitly returning a boolean

Reply:
nice catch